pyasic/miners/types/whatsminer/M3X/__init__.py

```diff
@@ -15,6 +15,8 @@
 # ------------------------------------------------------------------------------
 
 from .M30 import M30V10, M30V20
+from .M30K import M30KV10
```

Comment:
Make sure all new models get added to `pyasic/miners/miner_factory/MINER_CLASSES`